tests/specs/packages.js

```diff
@@ -0,0 +1,190 @@
+define(["dist/amd-loader"], function(AMDLoader) {
+  describe("Test suite", function() {
+    var bitloader;
+    beforeEach(function() {
+      bitloader = new AMDLoader({
+        baseUrl: "../",
+        packages: [
+          {
+            location: "tests",
+            main: "main",
+            name: "js"
+          }
+        ]
+      });
+    });
+
+    describe("When Importing a number", function() {
+      var number;
+      beforeEach(function() {
+        return bitloader.import("js/number.js")
+          .then(function (_number) {
+            console.log(arguments);
+            number = _number;
+          });
+      });
+
+      it("then the number is 3.14", function() {
+        expect(number).to.equal(3.14);
+      });
+    });
+
+    describe("When Importing an empty object", function() {
+      var result;
+      beforeEach(function() {
+        return bitloader.import("js/emptyobject.js")
+          .then(function (_result) {
+            result = _result;
+          });
+      });
+
+      it("then the result is an object", function() {
+        expect(result).to.be.an("object");
+      });
+
+      it("then the result is an object with no properties", function() {
+        expect(Object.keys(result).length).to.equal(0);
+      });
+    });
+
+    describe("When Importing two files", function() {
+      var emptyobject, emptystring;
+      beforeEach(function () {
+        return bitloader.import(["js/emptyobject", "js/emptystring"])
+          .then(function (_emptyobject, _emptystring) {
+            emptyobject = _emptyobject;
+            emptystring = _emptystring;
+          });
+      });
+
+      it("then the first result is one object", function() {
```

Comment:
The test isn't really testing quantity... It is testing type.  But something to think about is to being more concise with what you are testing.  The first result is an empty object and the second is a string.  So maybe the describe should be more along the lines of `When Importing an empty object and a string`.  Here is how I would probably wrap this:

``` javascript

describe("When importing an empty object and an empty string", function() {
     var emptyobject, emptystring;
     beforeEach(function () {
       return bitloader.import(["js/emptyobject", "js/emptystring"])
         .then(function (_emptyobject, _emptystring) {
           emptyobject = _emptyobject;
           emptystring = _emptystring;
         });
     });

    describe("and the empty object is loaded", function() {
      it("then `emptyobject` is an object", function() {});
      it("then `emptyobject` does not have properties", function() {});
    });

    describe("and the empty string is loaded", function() {
      it("then `emptystring` is a string", function() {});
      it("then `emptystring` has length 0", function() {});
    });
});
```

Something along those lines